packages/angular/cli/models/command.ts

```diff
@@ -101,6 +101,9 @@ export abstract class Command<T extends BaseCommandOptions = BaseCommandOptions>
       this.logger.info(`options:`);
       opts
         .filter(o => !o.hidden)
+        .filter((val, index, self) => {
```

Comment:
Ideally we fix the schemas rather than filtering the out what’s duplicate.

Reply:
Hey. Thanks a lot for reviewing. 
I couldn't figure out what in the schema is causing the duplicate output

Reply:
I actually think the proper fix might be in the [parseJsonSchemaToOptions](https://github.com/angular/angular-cli/blob/master/packages/angular/cli/utilities/json-schema.ts#L158) function. Having multiple times the same property isn't an error in JSON Schema world (as long as those properties match each others), but for the options you should only have one. So we should skips options we already found.

At that line, add a check for `if (options.some(o => o.name === name) { return; }` and you should be good. That way you fix the source of the problem and not the symptoms :)

Reply:
Understood.
I have updated the PR.